### PR TITLE
append button padding variables (fix issue #23550)

### PR DESCRIPTION
**Pull request by isychev**
- split padding variables $input-btn-padding
- create variables `$input-padding-y $input-padding-x $input-line-height` for all size
- create variables `$btn-padding-y $btn-padding-x $btn-line-height` for all size
- update all dependencies of variables
- format spaces offset in variables.scss

### CODE REVIEW

**Comment by mdo** [comment]
Something here is outta whack—the entire variables file has modifications.

**Comment by isychev** [comment]
@mdo 
I format spaces offset in variables.scss, only spaces
maybe I send pull request without offests?

**Comment by XhmikosR** [comment]
@isychev: please remove the formatting changes.

File: scss/_variables.scss [hunk 1]
```diff
@@ -170,141 +170,154 @@ $container-max-widths: (
...
 
 
 // Buttons
 //
 // For each of Bootstrap's buttons, define text, background and border color.
 
-$input-btn-padding-y:       .5rem !default;
-$input-btn-padding-x:       .75rem !default;
-$input-btn-line-height:     1.25 !default;
+$input-padding-y:       .5rem !default;
```
Comment (andresgalante) [inline scss/_variables.scss]: Can we move these input related variables under Forms on line 340 please?

**Pull request closed without merging**